lte/gateway/python/integ_tests/s1aptests/s1ap_wrapper.py

```diff
@@ -81,6 +81,12 @@ def __init__(
         self._s1_util = S1ApUtil()
         self._enBConfig(ip_version)
 
+        if os.environ.get("FED_MODE") == "True":
+            federated_mode = True
+            print("\n************************* Running the test in Federated Mode")
+        else:
+            print("\n************************* Running the test in Non-Federated Mode")
+
```

Comment:
Is that not a bit overly bloated? I think something along the following lines would be more concise.
```python
federated_mode = (os.environ.get("FED_MODE") == "True")
print(f"\n *** Running the test in {'Non-' if not federated_mode else ''}Federated Mode")
```